ui/public/index.html

```diff
@@ -24,20 +23,9 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>Weather Telemetry</title>
   </head>
   <body>
-    <noscript>You need to enable JavaScript to run this app.</noscript>
```

Comment:
Is there a reason why you don't want this tag?

Reply:
Oh I removed it accidentally. I would bring it back.